tox.ini

```diff
@@ -28,12 +28,17 @@ envlist =
 
 [testenv]
 commands = coverage run -p -m pytest --durations=20 {posargs:tests} -m "not slow"
+# use custom install command to install CPU versions of torch & PyG
+install_command =
+    python -m pip install {opts} {packages} --extra-index-url https://download.pytorch.org/whl/cpu --extra-index-url https://data.pyg.org/whl/torch-1.11.0+cpu.html
```

Comment:
does not work since PyG can only be installed *after* torch has already been installed.